src/components/search-results/search-results.tsx

```diff
@@ -1,20 +1,24 @@
 import React, { FC, useEffect } from "react";
+import CardComponent from "../card/card";
+import { SearchResult } from "../../models/request/search";
+import Grid from "@mui/joy/Grid";
 
 interface SearchResultsProps {
     search: string;
-    results: any[];
+    results: SearchResult[];
 }
 export const SearchResults: FC<SearchResultsProps> = ({ search, results }) => {
     return (
-        <div>
-            <p>
-                Searching for: <b>{search}</b>
-            </p>
-            <ul>
-                {results.map((result, i) => (
-                    <li key={i}>{result}</li>
-                ))}
-            </ul>
-        </div>
+        <Grid container spacing={4}>
+            <Grid xs={12}>
+                <h1>Search results for &quot;{search}&quot;</h1>
+            </Grid>
```

Comment:
It might be out of scope, but, right now, when results are loading and you search somthing else or change the page, the load icon appear and makes everything shifts. Maybe, we could put it in the same line as this text so the layout of the page is not affected.

Reply:
I will try to check this after everything is done